roboteam_ai/src/control/ballHandling/BallHandlePosControl.cpp

```diff
@@ -78,9 +82,17 @@ RobotCommand BallHandlePosControl::getRobotCommand(const RobotPtr &r, const Vect
     // if we do not have the ball yet, go get it
     bool robotDoesNotHaveBall = ! robot->hasBall();
     bool robotIsTooFarFromBall = (robot->pos - ball->pos).length2() > maxBallDistance*maxBallDistance;
+    bool robotIsTouchingBall =
+            (robot->pos - ball->pos).length() < (Constants::ROBOT_RADIUS() + Constants::BALL_RADIUS())*1.05;
```

Comment:
magic number

Reply:
I refactored all of constants.h, but I fixed it.